datatypes.md

```
@@ -136,6 +136,18 @@ $ redis-cli GRAPH.QUERY G "MATCH (n) RETURN n {.name, .age} AS projection"
 2) 1) 1) "{name: Jeff, age: 32}"
 ```
 
+#### Map merging
+
+You can combine two maps, where values in the second map will override corresponding values in the first map.
+For example:
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)